packages/appeal-reply-service-api/src/controllers/reply.js

```diff
@@ -83,17 +83,20 @@ module.exports = {
         await mongodb
           .get()
           .collection(dbId)
-          .updateOne({ _id: idParam }, { $set: { uuid: idParam, reply: newDoc } })
-          .then(() => {
+          .updateOne({ _id: idParam }, { $set: { uuid: idParam, reply: req.body } })
+          .then(async () => {
             logger.debug(`Updated reply ${idParam}\n`);
-
-            if (isFirstSubmission) {
-              logger.info({ newDoc }, 'STEVE: api');
-              queue.addAppealReply(newDoc);
-              // TODO: call to notify will go here
+            if (req.body.state === 'SUBMITTED') {
+              try {
+                await notify.sendAppealReplySubmissionConfirmationEmailToLpa(req.body);
+                res.status(200).send(req.body);
+              } catch (err) {
+                logger.error(`Problem sending email ${idParam}\n${err}`);
```

Comment:
`logger.error({err, idParam}, `Problem sending email`);`

or afaik, you will lose the correlation id